2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -19,7 +19,33 @@
 //cspell: enable
 
 function createBookList(books) {
-  // your code goes in here, return the ul element
+  const ul = document.createElement('ul');
+  for (const item of books) {
+    const newPar = document.createElement('p');
+    newPar.innerText = `${item.title} - ${item.author}`;
+    const li = document.createElement('li');
+    li.appendChild(newPar);
+
+    const img = document.createElement('img');
+    if (item.title === 'The Design of Everyday Things') {
```

Comment:
maybe it is better to have this logic outside of this function and to add property of imageSource to your books array. That way you can reuse these src properties if you use books constant somewhere else